docs/source/contributing/development-environment.rst

```diff
@@ -57,39 +57,30 @@ Substitute ``cargo build`` with ``cargo test`` to test, or ``cargo check`` to ru
 
 In the above commands, the features ``untrusted`` and ``bindings`` are enabled.
 
-Setting a feature changes how the crate compiles:
-
-
-.. dropdown:: Feature List
-
-    .. list-table::
-        :widths: 25 75
-        :header-rows: 1
-
-        * - Name
-          - Description
-        * - ``untrusted``
-          - Enables untrusted features ``contrib`` and ``floating-point``.
-        * - ``contrib``
-          - Enable to include constructors that have not passed the vetting process.
-        * - ``honest-but-curious``
-          - Enable to include constructors whose differential privacy (or stability) properties
-            rely on the constructor arguments being correct.
-            That is, if a user/adversary is 'honest' in specifying the constructor arguments,
-            then even if they later become 'curious' and try to learn something from the measurement outputs,
-            they will not be able to violate the differential privacy promises of the measurement.
-        * - ``floating-point``
-          - Enable to include transformations and measurements with floating-point vulnerabilities.
-        * - ``bindings``
-          - Enable to generate Python and R source code. Depends on the ``ffi`` and ``derive`` features. 
-        * - ``partials``
-          - Enable to generate ``then_*`` functions from the corresponding ``make_*`` functions. Depends on the ``derive`` feature.
-        * - ``ffi``
-          - Enable to include C foreign function interfaces. Implicit in the ``bindings`` feature.
-        * - ``derive``
-          - Enable to support code generation and links to proofs in documentation. Implicit in the  ``bindings`` and ``partials`` features.
-        * - ``use-openssl``
-          - Already enabled. Use OpenSSL for secure noise generation.
+Setting a feature changes how the crate compiles. 
+In addition to the :ref:`feature-listing`, Rust also has the following features:
+
+.. _rust-feature-listing:
+
+.. dropdown:: Rust-Exclusive Feature List
+
+    
+   .. list-table::
+      :widths: 25 75
+      :header-rows: 1
+
+      * - ``untrusted``
```

Comment:
```suggestion
      * - Name
        - Description
      * - ``untrusted``
```
The first row is bolded.

I'd make this a full list, rather than just a subset. Slightly redundant, but good to have it all in one place.